api/create_materialized_view.md

```diff
@@ -45,8 +45,14 @@ parameter. For more information, see
 
 Continuous aggregates have some limitations of what types of queries they can
 support, described in more length below. For example, the `FROM` clause must
-provide only one hypertable or underlying continuous aggregate, and joins, CTEs, or subqueries are not
-supported. The `GROUP BY` clause must include a time bucket on the underlying
+provide only one hypertable or underlying continuous aggregate, CTEs, or subqueries are not
+supported. The `FROM` clause now supports JOINS but it has to adhere to the following requirements,
+1. The JOIN can be between only two tables, where one is a hypertable (and NOT a continuous aggregate itself)
+   and other table is a normal PostgreSQL table
+2. JOIN type is INNER
+3. JOIN conditions is equality.
```

Comment:
@RafiaSabih can you please check and point out any issues with my proposed update? I also added that we don't track the regular PG table for invalidations

```suggestion
1. Only joins between two tables are currently supported, where one is a hypertable and the other table is a normal PostgreSQL table. The order of tables in the `JOIN` clause does not matter.
2. Only the `INNER` `JOIN` type is currently supported.
3. Only equality conditions are currently supported as `JOIN` conditions.
4. Joins in hierarchical continuous aggregates are not currently supported. The hypertable on the `JOIN` condition must be a normal hypertable, not a continuous aggregate itself.
5. Changes on the normal PostgreSQL table are not tracked. Only changes on the hypertable result in updates to the continuous aggregate when it is refreshed.
```